### PR TITLE
Minor changes added to the pipeline

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] If necessary, also make a PR on the [nf-core/proteomicslfq branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/proteomicslfq)

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -144,7 +144,10 @@ try {
 }
 
 profiles {
-  conda { process.conda = "$baseDir/environment.yml" }
+  conda {
+    process.conda = "$baseDir/environment.yml"
+    conda.createTimeout = '1 h'
```
Comment (Zethson) [inline nextflow.config:149]: Just out of interest. Why is this required?
I never had the creation of a Conda env fail.

File: main.nf [hunk 2]
```diff
@@ -121,6 +121,7 @@ def helpMessage() {
       --protein_level_fdr_cutoff    Protein level FDR cutoff (this affects and chooses the peptides used for quantification)
 
       Quantification:
+      --quantification_method       Quantification method supported by proteomicslfq ('feature_intensity' or 'spectral_counting', default: 'feature_intensity')
```
Comment (jpfeuffer) [inline main.nf:124]: This also needs to be added to the nextflow.config with a default value:
`params.quantification_method = "feature_intensity"` and in the nextflow_schema.json under the Quantification group with a help text and the allowed strings.
And you need to actually use it in the "proteomicslfq" process with the "params.quantification".
If you want you could add this with a test config for CI as well.

File: conf/base.config [hunk 3]
```diff
@@ -16,7 +16,7 @@ process {
   memory = { check_max( 8.GB * task.attempt, 'memory' ) }
   time = { check_max( 4.h * task.attempt, 'time' ) }
 
-  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
+  errorStrategy = { task.exitStatus in [140,143,137,104,134,139] ? 'retry' : 'finish' }
```
Comment (jpfeuffer) [inline conf/base.config:19]:
```suggestion
  // nf-core defaults plus 140 (LSF out-of-time)
  errorStrategy = { task.exitStatus in [140,143,137,104,134,139] ? 'retry' : 'finish' }
```

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: nextflow.config (hunk 1 above)
Comment (ypriverol) [reply nextflow.config:149]: If the file system is share file system it will fail. At least in our nfs file system for the number of dependencies we have it fails.

File: main.nf (hunk 2 above)
Comment (ypriverol) [reply main.nf:124]: - [x] is already added in the proteomicslfq (https://github.com/nf-core/proteomicslfq/blob/30cf776a7e622a962ae5fc2860a3853d8518acbd/main.nf#L1070). 
- [x] It is already in the nextflow.config (https://github.com/nf-core/proteomicslfq/blob/30cf776a7e622a962ae5fc2860a3853d8518acbd/nextflow.config#L93) 
- [ ] just added to the json (https://github.com/nf-core/proteomicslfq/pull/91/commits/fd3185cb28e855d4922f800ebec79f55890911db#diff-2d9d6ae344b889545d73f7c118b3f05aR695)

**Comment by ypriverol** [comment]
@julianus changes have been done !!!

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: main.nf [hunk 4]
```diff
@@ -1061,8 +1062,28 @@ process proteomicslfq {
      file "*.log"
 
     script:
-     """
-     ProteomicsLFQ -in ${(mzmls as List).join(' ')} \\
+     if(params.quantification_method == "spectral_counting")
```
[the remark this answers is not in the document]
Comment (ypriverol) [reply main.nf:1065]: What I mention before, I don't know if you can pass the script empty parameter?

[the remark this answers is not in the document]
Comment (jpfeuffer) [reply main.nf:1065]: yes you can: https://github.com/nf-core/proteomicslfq/blob/11ac18c87f81b4cf802356deecfdd7f5e52ee2b5/main.nf#L660

[the remark this answers is not in the document]
Comment (ypriverol) [reply main.nf]: done

File: nextflow_schema.json [hunk 5]
```diff
@@ -694,14 +694,13 @@
                 },
                 "quantification_method": {
                     "type": "string",
-                    "description": "Currently UNSUPPORTED in this workflow. Choose between feature-based quantification based on integrated MS1 signals ('feature_intensity'; default) or spectral counting of PSMs (spectral_counting).",
+                    "description": "Choose between feature-based quantification based on integrated MS1 signals ('feature_intensity'; default) or spectral counting of PSMs (spectral_counting).",
```
Comment (jpfeuffer) [inline nextflow_schema.json:697]:
```suggestion
                    "description": "Choose between feature-based quantification based on integrated MS1 signals ('feature_intensity'; default) or spectral counting of PSMs ('spectral_counting'). WARNING: 'spectral_counting' is not compatible with our MSstats step yet and will be disabled automatically",
```

**Review by jpfeuffer — APPROVED** [review]
